persistence-dse-6.8/pom.xml

```
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <!-- inherit common settings -->
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.11-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.db.dse</groupId>
+  <artifactId>persistence-dse-6.8</artifactId>
+  <properties>
+    <dse.version>6.8.2-69139b3</dse.version>
+    <junit.version>5.6.2</junit.version>
+  </properties>
+  <repositories>
+    <repository>
+      <id>central</id>
+      <url>https://repo.maven.apache.org/maven2</url>
+    </repository>
+    <repository>
+      <id>artifactory-snapshots</id>
+      <url>https://repo.datastax.com/datastax-snapshots-local</url>
+    </repository>
+    <repository>
+      <id>artifactory-releases</id>
+      <url>https://repo.datastax.com/datastax-releases-local</url>
+    </repository>
+  </repositories>
+  <dependencies>
+    <dependency>
+      <groupId>io.stargate.db</groupId>
+      <artifactId>persistence-common</artifactId>
+      <version>0.0.9</version>
```

Same updates in this file
```suggestion
      <version>0.0.11-SNAPSHOT</version>
```